float32 steeringAngle
float32 speed